src/adapters/lokijs/worker/lokiExtensions.js

```diff
@@ -1,39 +1,79 @@
 // @flow
+/* eslint-disable no-undef */
 
 import Loki, { LokiMemoryAdapter } from 'lokijs'
-import LokiIndexedAdapter from 'lokijs/src/loki-indexed-adapter'
 
-export function newLoki(name: ?string, peristenceAdapter?: LokiMemoryAdapter): Loki {
-  const newAdapter =
-    process.env.NODE_ENV === 'test' ? new LokiMemoryAdapter() : new LokiIndexedAdapter(name)
+const isIDBAvailable = () => {
+  return new Promise(resolve => {
+    // $FlowFixMe
+    if (typeof indexedDB === 'undefined') {
+      resolve(false)
+    }
 
-  return new Loki(name, {
-    adapter: peristenceAdapter || newAdapter,
-    autosave: true,
-    autosaveInterval: 250, // TODO: Remove this and force database save when we have transactions
-    env: 'BROWSER', // TODO: ?
-    verbose: true, // TODO: remove later
```

Comment:
Are these comments no longer valid? I don't see them in new version :)

Reply:
verbose doesn't really do much of anything - IIRC it only spits out errors to the console, and env is not necessary / doesn't do anything if you pass adapter explicitly

Reply:
Cool, just wanted to make sure these weren't omitted by mistake :)